generator/gen_bindings.go

```diff
@@ -546,16 +546,8 @@ func (gen *Generator) proxyArgFromGo(memTip tl.Tip, name string,
 		proxy = fmt.Sprintf("%s(%s)", helper.Name, name)
 		return proxy, helper.Nillable
 	case isPlain && goSpec.Slices != 0: // ex: []byte
-		gen.submitHelper(sliceHeader)
```

Comment:
this looks like its checking if the "sliceHeader" has any dependencies, and if so triggering them to be generated as well, I'm not sure we want to disable that?